mythx_cli/util.py

```diff
@@ -10,8 +11,134 @@
 LOGGER = logging.getLogger("mythx-cli")
 
 
+class SourceMapLocation:
+    def __init__(
+        self, offset: int = 0, length: int = 0, file_id: int = -1, jump_type: str = "-"
+    ):
+        self.o = int(offset)
+        self.l = int(length)
+        self.f = int(file_id)
+        self.j = jump_type
+
+    @property
+    def offset(self) -> int:
+        return self.o
+
+    @offset.setter
+    def offset(self, value: int) -> None:
+        value = int(value)
+        if value <= 0:
```

Comment:
Question: do we really restrict assigning `0` for offset?

Reply:
@dmuhs made this so we'll have to wait for the reply